suite-native/helpers/src/hooks/useCopyToClipboard.tsx

```diff
@@ -7,8 +8,10 @@ export function useCopyToClipboard() {
     const copyToClipboard = useCallback(async (value: string, toastMessage?: string) => {
         await Clipboard.setStringAsync(value);
 
-        // TODO: Replace with our own notification UI when ready.
-        Toast.show(toastMessage ?? 'Copied to clipboard.');
+        if (Platform.OS === 'ios') {
```

Comment:
would be nice to have comment like "Android is showing native toast when copying text" or smthing like that.

Reply:
fixed